src/js/helpers/accountHelper.js

```diff
@@ -38,7 +38,7 @@ export const fetchAvailableObjectClasses = (federalAccountId) => apiRequest({
 });
 
 export const fetchAllSubmissionDates = () => apiRequest({
-    url: 'v2/references/submission_periods/'
+    url: 'v2/references/submission_periods/?use_cache=true'
```

Comment:
Outstanding question to @collinwr of whether we want to add this param for the Covid profile only